dev/devenv-readme.md

```diff
@@ -12,49 +12,175 @@ First off, if you're on one of the following platforms, we provide the [devenv-i
 It can be run as follows:
 
 1. Install Python 3.
-    * On Windows, this can be done using [apt-cyg](https://github.com/transcode-open/apt-cyg), as follows:
-    
-        ```
-        $ apt-cyg install python3 python3-setuptools
-        ```
+
+  On Windows, this can be done using [apt-cyg](https://github.com/transcode-open/apt-cyg), as follows:
     
+    ```
+    $ apt-cyg install python3 python3-setuptools
+    ```
+
+    __Note:__ If apt-cyg is having problems connecting to your cygwin mirror this may be due to an incorrect HOSTTYPE setting.  Verify your HOSTTYPE does not have additional decoration(i.e. x86_64-cygwin) and only contains the your system architecture(i.e. x86_64) that you are attempting to install on.
+    ```
+    # Example incorrect setting
+    $ echo $HOSTTYPE
+    x86_64-cygwin
+
+    # Example correct setting
+    $ echo $HOSTTYPE
+    x86_64
+    ```
+    HOSTTYPE can be overridden on the command line or set in your shell configuration(i.e. .bashrc, .cshrc, etc).  To override HOSTTYPE on the command line use the following construct:
+    ```
+    # for csh/tcsh
+    > setenv HOSTTYPE x86_64 && apt-cyg <command>
+
+    # for bash
+    $ HOSTTYPE=x86_64 apt-cyg <command>
+    ```
 1. Run the script:
 
-    ````
+    ```
     $ ./devenv-install.py
-    ````
+    ```
 
 What does it do for you? Great question! It will create a `~/workspaces/tools/` directory and then download and install (as a user) the following into there for you:
 
 * An [Oracle Java 8 JDK](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
-* [Eclipse Mars](https://eclipse.org/mars/)
-    * These Eclipse Plugins:
-        * [m2e-apt](https://developer.jboss.org/en/tools/blog/2012/05/20/annotation-processing-support-in-m2e-or-m2e-apt-100-is-out?_sscc=t): Allows Maven projects in Eclipse to easily leverage [the JDK's Annotation Processing framework](http://docs.oracle.com/javase/7/docs/technotes/guides/apt/).
 * [Apache Maven](https://maven.apache.org/)
+* [Latest Eclipse Version](https://www.eclipse.org/downloads/)
+* Eclipse __m2e-apt__ Plugin
+    * The m2e-apt plugin allows Maven projects in Eclipse to easily leverage [the JDK's Annotation Processing framework](http://docs.oracle.com/javase/7/docs/technotes/guides/apt/).
+    * Install m2e-apt using the [Drag To Install](http://marketplace.eclipse.org/content/m2e-apt) option or install from within the Eclipse IDE by opening **Help > Eclipse Marketplace...** and searching for **m2e-apt** using the **Eclipse Marketplace** find dialog box.  Click the **Install** button for the plugin and restart Eclipse when prompted. 
+    * [Usage Instructions for m2e-apt](https://developer.jboss.org/en/tools/blog/2012/05/20/annotation-processing-support-in-m2e-or-m2e-apt-100-is-out?_sscc=t)
 
 If you're not using one of those supported platforms, or would prefer to setup things manually, you'll want to download and install the items listed above yourself.
 
+## Cygwin Configuration
```

Comment:
You added some Cygwin-related instructions up above (the `HOSTTYPE` stuff). Does that flow okay, or do things need to be consolidated a bit?

Reply:
That was placed there in case they have apt-cyg problems like I did.  I think it should stay as they might not get any further in the document if they get stuck on apt-cyg.

Reply:
I thought about it more, I can move it and just provide a link in the apt-cyg section so it flows a little better.